recipes/examl/build.sh

```diff
@@ -0,0 +1,8 @@
+#! /bin/bash
+set -x
+set -e
+
+cd parser && make -f Makefile.SSE3.gcc && mv parse-examl ${PREFIX}/bin
+cd ../examl && make -f Makefile.SSE3.gcc && mv examl ${PREFIX}/bin
+rm -f *.o && make -f Makefile.AVX.gcc && mv examl-AVX ${PREFIX}/bin
+rm -f *.o && make -f Makefile.OMP.AVX.gcc && mv examl-OMP-AVX ${PREFIX}/bin
```

Comment:
@daler well probably not. The installation instructions (from 2016) mention "recently purchased system". AVX has been around since 2011 so I'm hoping the travis system would work. Since I've been disconnected from work quite a while, I just noticed that the build deployment has changed. One evident change is that the output of the tests isn't shown in the logs, right? 

The osx SUBDAG=1 build currently fails, complaining about an unsupported clang option '-fmopen'. mpicc should use gcc which apparently seems to be out of date. Maybe adding a dependency on a newer gcc will help.